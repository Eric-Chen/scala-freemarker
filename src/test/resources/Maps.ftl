[#ftl]
${map.one} ${map.two.one} ${map.two.two}

[#list map?keys?sort as k]${k}[#if k_has_next], [/#if][/#list]