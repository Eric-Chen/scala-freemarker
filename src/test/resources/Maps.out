Hello to Joe Smith

one, two